### DISCUSSION TITLE
Request for Assistance with GitHub Education Account - Remote Learning Situation due to the ongoing war in Ukraine

### DISCUSSION THREAD

**Discussion by yelyzaveta135** [community/community#76025] [GitHub Education]
### Hi everyone,

due to the unfortunate circumstances of the war in Ukraine, I am currently studying remotely. However, I have encountered a challenge in updating my academic affiliation documentation to reflect this change in my learning environment. Unfortunately, I was unable to make this adjustment during the ongoing conflict.

The system is indicating that I am not near any campus location for the school I have selected, and I understand that my academic affiliation documentation must indicate virtual learning if I am engaging in remote education. Given the exceptional circumstances in my region, updating this documentation has proven to be a complex task.

I kindly request your guidance on how to proceed in this situation. Is there a process or alternative method I can follow to ensure that my GitHub Education account accurately reflects my current mode of study? I am committed to complying with GitHub's policies and requirements and appreciate any assistance you can provide.

Additionally, if there are steps I can take or documentation I can provide to further explain the challenges I am facing, please let me know. I am eager to resolve this matter promptly and appreciate your understanding during these challenging times.

**Comment by SAINT-CHRIS** [comment]
This is serious and we need help
